packages/firestore/src/model/document.ts

```diff
@@ -101,6 +101,13 @@ export interface Document {
    */
   readonly readTime: SnapshotVersion;
 
+  /**
+   * The timestamp at which the document was created. This value increases
```

Comment:
Is this comment copied from proto? Looks pretty thorough.

Reply:
Yes, copied from the proto.